<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomItem</class>
 <widget class="QWidget" name="RoomItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>632</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget_3  {
	border: 1px solid rgb(224, 224, 224);
}
QWidget {
	background-color: #fff;
	border-radius: 8px;
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>5</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>0</y>
         <width>151</width>
         <height>151</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource.qrc">:/img/img/exam.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>7</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QLabel" name="nameLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>44</y>
         <width>341</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Source Sans 3</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <stylestrategy>PreferAntialias</stylestrategy>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string>Introduction to Artificial Intelligence</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>81</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Source Sans 3</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">border:1px solid rgb(39, 174, 96);
color: rgb(39, 174, 96);
border-radius: 8px;</string>
       </property>
       <property name="text">
        <string>Lobby</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>85</y>
         <width>28</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource.qrc">:/img/img/group.svg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>89</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource.qrc">:/img/img/clock.svg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="capacityLabel">
       <property name="geometry">
        <rect>
         <x>42</x>
         <y>84</y>
         <width>30</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Source Sans 3</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>50</string>
       </property>
      </widget>
      <widget class="QLabel" name="timeLimitLabel">
       <property name="geometry">
        <rect>
         <x>105</x>
         <y>84</y>
         <width>41</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Source Sans 3</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>75m</string>
       </property>
      </widget>
      <widget class="QPushButton" name="joinPushButton">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>105</y>
         <width>98</width>
         <height>36</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Source Sans 3</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">#joinPushButton {
	background-color: rgb(0, 191, 154);
	border-radius: 4px;
	color: #fff;
	border: none;
}
#joinPushButton:hover {
	border: 1px solid rgb(0, 191, 154);
	color: rgb(0, 191, 154);
	background: none;
}</string>
       </property>
       <property name="text">
        <string>Join</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>120</y>
         <width>18</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource.qrc">:/img/img/calendar.svg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="startTimeLabel">
       <property name="geometry">
        <rect>
         <x>42</x>
         <y>117</y>
         <width>171</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Source Sans 3</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>2023-01-01 17:00:00</string>
       </property>
      </widget>
      <widget class="QLabel" name="lockLabel">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>10</y>
         <width>22</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource.qrc">:/img/img/lock-red.svg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
